extensions/amp-story/0.1/progress-bar.js

```diff
@@ -98,6 +105,14 @@ export class ProgressBar {
     return this.getRoot();
   }
 
+  /**
+   * create mapping of pageIds to position in progress bar
+   * @param {Array} pages
+   */
+  makePageIdMap_(pages) {
+    pages.forEach((page, i) => this.pageIdMap_[page.element.id] = i);
```

Comment:
nit: you can inline this one line function

Reply:
moved